### ISSUE TITLE
Some Ideas. [Suggestions]

### ISSUE THREAD

**Issue by Handeril** [Reactions-Modpack/VoidIslandControl#1]
1. A simple check for if the worldtype is loaded when running the commands might be a good idea - I noticed that was a bug on my modpack (in the early days of me trying to add a Skyblock mode, I used the original version of Void Island Control to try to add a Skyblock Mode to my modpack). The commands could still be run in normal worlds (I believe), and I think even default spawn islands spawned in normal worlds whether it was "Void?" or not, however, that second claim will need further testing, I went through 3 to 5 Skyblock Map mods while setting up that option.

2. Need to make sure Non-Void End/Nether dimensions generate when the world is not the "Void?" world type. I think this was a bug I ran into during that testing phase.

3. The starter items may also need a check for the "Void?" worldtype as well, but I'm not sure.

4. Option to disable BuildProtection/Anti-Griefing Measures via configuration, since other Anti-Griefing/BuildProtection mods or measures may be in use.

5. Finally, possibly replacing the system of the configured "Start Island" in the files and commands, to being a "Customize" option for the "Void?" worldtype. With a "sanity check" to see if there's more than one island enabled before showing the button. **However, the configuration option could be repurposed to being the "server starter island", for ease of use on multiplayer servers.**

**Comment by CaliforniaDemise** [comment]
Yeah, great suggestions. When i forked it i never thought someone might want to add skyblock type to their pack.
Also customizable starter island sounds cool too.
